cloudbuild.yaml

```diff
@@ -16,4 +16,6 @@ steps:
     - '--region=${_REGION}'
     - '--platform=managed'
     - '--allow-unauthenticated'
-    - '--image=eu.gcr.io/${PROJECT_ID}/${_SERVICE_NAME}:$SHORT_SHA'
\ No newline at end of file
+    - '--image=eu.gcr.io/${PROJECT_ID}/${_SERVICE_NAME}:$SHORT_SHA'
```

Comment:
Been a while since I looked at this code but should we drop the "eu." ?  Not necessary if you aren't sure either since it seems to be working.

Reply:
From my understanding, the "eu." is there to trigger an error in the build so that the user can fix the issue in the assessment.

Reply:
Makes sense, thanks